src/arrays.jl

```diff
@@ -14,6 +14,11 @@ function Base.getindex(u::UniFinArr{<:Any,<:Any,R,P,N},
     return UnivariateFinite(u.scitype, u.decoder, prob_given_ref)
 end
 
+function Base.getindex(u::UniFinArr, idx::CartesianIndex)
```

Comment:
add missing definition. Now things like `Matrix` or `copy` will work.

Reply:
Does this address https://github.com/JuliaAI/CategoricalDistributions.jl/issues/43

Reply:
No, it addresses the first bullet in https://github.com/JuliaAI/CategoricalDistributions.jl/issues/41.
#43 is separate, but it does not have to be resolved in this PR as it is unrelated. That is why I have opened a separate issue.